examples/13_write_dynamic_configuration.cpp

```diff
@@ -0,0 +1,140 @@
+#include <openPMD/openPMD.hpp>
+
+#include <algorithm>
+#include <iostream>
+#include <memory>
+#include <numeric> // std::iota
+
+using std::cout;
+using namespace openPMD;
+
+bool hasAdios2()
+{
+    auto variants = getVariants();
+    auto iterator = variants.find( "adios2" );
+    return iterator != variants.end() && iterator->second;
+}
+
+int main()
+{
+    if( !hasAdios2() )
```

Comment:
We intentionally always contain all known keys, so you can simplify to:
```suggestion
    if( !getVariants()["adios2"] )
```
if you like.

Reply:
Ah, thanks :)
I was initially checking only for key existence and kept wondering why the CI was crashing..